src/pages/ModelOfferPage.tsx

```diff
@@ -39,12 +42,20 @@ const ModelOfferPage = () => {
     postApplication();
   };
 
+  const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
+  const handleOpenModal = () => {
+    setIsModalOpen(true);
+  };
+  const handleCloseModal = () => {
+    setIsModalOpen(false);
+  };
```

Comment:
```suggestion
  const handleModal = () => {
      setIsModalOpen((prev) => !prev);
  };
```
이렇게 안한 이유는 leftBtnFn과 rightBtnFn에 함수이름을 더 확실하게 하기 위함일까용?!
아니라면 더 간결하게 할 수 있을 것 같습니당 👍🏻